Cannot install tools due to nodegit install failing

Attempting to install via `npm install -g @mapbox/github-release-tools` results in the following error when trying to install `nodegit`:

```
~ julianrex$ npm install nodegit

> nodegit@0.22.1 install /Users/julianrex/node_modules/nodegit
> node lifecycleScripts/preinstall && node lifecycleScripts/install

[nodegit] Running pre-install script
[nodegit] Configuring libssh2.
[nodegit] Running install script
node-pre-gyp
ERR! Tried to download(404): https://axonodegit.s3.amazonaws.com/nodegit/nodegit/nodegit-v0.22.1-node-v64-darwin-x64.tar.gz 
node-pre-gyp ERR! Pre-built binaries not found for nodegit@0.22.1 and node@10.1.0 (node-v64 ABI, unknown) (falling back to source compile with node-gyp)
```

The source compilation eventually fails with compile errors (and warnings), for example:

```
CC(target) Release/obj.target/libssh2/vendor/libssh2/src/openssl.o
../vendor/libssh2/src/openssl.c:71:11
: error: incomplete definition of type 'struct rsa_st'
    (*rsa)->e = BN_new();
    ~~~~~~^

[lines deleted]

npm ERR! Failed at the nodegit@0.22.1 install script.

[lines deleted]
```

--- a/package.json
+++ b/package.json
@@ -14,7 +14,6 @@
     "bottleneck": "^2.3.0",
     "gitconfiglocal": "^2.0.1",
     "latest-semver": "^1.0.0",
-    "nodegit": "^0.22.0",
     "parse-github-url": "^1.0.2",
     "prepend-file": "^1.3.1",
     "yargs": "^11.0.0"

--- a/src/backport.js
+++ b/src/backport.js
@@ -14,7 +14,7 @@ module.exports = async function ({change, branch}) {
     const prReviews = (await octokit.pullRequests.getReviews({owner, repo, number})).data;
     const prPatch = (await octokit.pullRequests.get({owner, repo, number, headers: {accept: 'application/vnd.github.v3.patch'}})).data;
 
-    execSync(`git branch backport-${number} ${branch}`);
+    execSync(`git branch backport-${number} origin/${branch}`);
     execSync(`git checkout backport-${number}`);
     execSync(`git am`, {input: prPatch});
     execSync(`git push origin --set-upstream backport-${number}`);

--- a/src/backport.js
+++ b/src/backport.js
@@ -1,6 +1,5 @@
 const parse = require('parse-github-url');
 const octokit = require('@octokit/rest')();
-const git = require('nodegit');
 const {execSync} = require('child_process');
 
 module.exports = async function ({change, branch}) {
@@ -15,10 +14,8 @@ module.exports = async function ({change, branch}) {
     const prReviews = (await octokit.pullRequests.getReviews({owner, repo, number})).data;
     const prPatch = (await octokit.pullRequests.get({owner, repo, number, headers: {accept: 'application/vnd.github.v3.patch'}})).data;
 
-    const repository = await git.Repository.open('.');
-    await repository.createBranch(`backport-${number}`, await repository.getBranchCommit(branch));
-    await repository.checkoutBranch(`backport-${number}`);
-
+    execSync(`git branch backport-${number} ${branch}`);
+    execSync(`git checkout backport-${number}`);
     execSync(`git am`, {input: prPatch});
     execSync(`git push origin --set-upstream backport-${number}`);
 

